app/frontend/package.json

```diff
@@ -40,6 +40,17 @@
       "prettier/@typescript-eslint",
       "prettier/react"
     ],
+    "plugins": [
+      "eslint-plugin-simple-import-sort"
```

Comment:
```suggestion
      "simple-import-sort"
```
Apparently you can omit the `eslint-plugin` bit according to their README

Reply:
I'm going to skip it for now so you don't have to re-approve, feel free to change it upstream with another PR.